alvr/server/cpp/platform/linux/EncodePipelineSW.cpp

```diff
@@ -88,7 +88,9 @@ alvr::EncodePipelineSW::EncodePipelineSW(Renderer *render, VkFrame &input_frame,
   encoder_ctx->max_b_frames = 0;
   encoder_ctx->gop_size = 0;
   encoder_ctx->bit_rate = settings.mEncodeBitrateMBs * 1000 * 1000;
+  encoder_ctx->rc_min_rate = encoder_ctx->bit_rate;
   encoder_ctx->rc_max_rate = encoder_ctx->bit_rate;
+  encoder_ctx->rc_buffer_size = encoder_ctx->bit_rate / settings.m_refreshRate;
```

Comment:
This destroys image quality with x264.
I'm actually thinking it would be better to left it at default value instead, as it doesn't seem to make a difference with VAAPI anyway.

Reply:
That's odd, it could be stuttery due to this setting but quality shouldn't be affected. Normally we need buffer size to be equal to size of the frame (maybe also multiplied by 1.1) for low latency video.

Reply:
Indeed, maybe it's related to the all-idr frames workaround.